contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeListItem/index.vue

```diff
@@ -48,13 +54,12 @@
         >
           {{ subtitle }}
         </VListTileSubTitle>
-        <p
+        <ToggleText
```

Comment:
What's the purpose of changing this?

Reply:
This handles descriptions more gracefully than just a regular <p> tag. As it is now, if any items in the main node panel have a decsription, the panel expands and blocks out the hierarchy and resource side panels